src/LiveDevelopment/LiveDevMultiBrowser.js

```diff
@@ -800,6 +806,22 @@ define(function (require, exports, module) {
     }
 
     /**
+     * Sets the current browser launcher mechanism to be used by live development
+     * (e.g., default browser, iframe-based browser, etc.)
+     * The launcher must provide the following method:
+     *
+     * - launch(url): Launch the given URL in the appropriate browser.
+     *
+     * @param {{launch: function(string)}} launcher
+     */
+    function setLauncher(launcher) {
+        if (!(launcher && launcher.launch)) {
+            throw new Error("LiveDevMultiBrowser.setLauncher(): launcher must have `launch` method");
```

Comment:
We'd normally log an error instead of throwing it. However, it may still make sense to have an error to be able to trace back the origin of the call. Like so (note the commas, not concatenation):

```javascript
console.log("Invalid launcher object: ", launcher, new Error("LiveDevMultiBrowser.setLauncher()");
```